### ISSUE TITLE
Prism failing with 3.0.0-rc.1

### ISSUE THREAD

**Issue by johnnyreilly** [facebook/docusaurus#9459]
### Have you read the Contributing Guidelines on issues?

- [X] I have read the [Contributing Guidelines on issues](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#issues).

### Prerequisites

- [X] I'm using the latest version of Docusaurus.
- [X] I have tried the `npm run clear` or `yarn clear` command.
- [X] I have tried `rm -rf node_modules yarn.lock package-lock.json` and re-installing packages.
- [ ] I have tried creating a repro with https://new.docusaurus.io.
- [ ] I have read the console error message carefully (if applicable).

### Description

I've just upgraded to 3.0.0-rc.1 and I'm getting Prism errors: https://github.com/johnnyreilly/blog.johnnyreilly.com/actions/runs/6658477531/job/18095419680

```bash
[success] [webpackbar] Client: Compiled with some errors in 1.74m
Error:  Client bundle compiled with errors therefore further build is impossible.
Module parse failed: Unexpected token (60:21)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
| 						],
| 						'operator': /\\$/m,
> 						'punctuation': /=/
| 					}
| 				},

--------------------------

Module parse failed: Unexpected token ([76](https://github.com/johnnyreilly/blog.johnnyreilly.com/actions/runs/6658477531/job/18095419680#step:5:77)3:929)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|     return "(?:" + r + "|" + s + ")";
|   })), lookbehind: true, greedy: true, alias: "atrule" }, directive: { pattern: /(^[ \t]*)%.+/m, lookbehind: true, alias: "important" }, datetime: { pattern: i(/\d{4}-\d\d?-\d\d?(?:[tT]|[ \t]+)\d\d?:\d{2}:\d{2}(?:\.\d*)?(?:[ \t]*(?:Z|[-+]\d\d?(?::\d{2})?))?|\d{4}-\d{2}-\d{2}|\d\d?:\d{2}(?::\d{2}(?:\.\d*)?)?/.source), lookbehind: true, alias: "number" }, boolean: { pattern: i(/false|true/.source, "i"), lookbehind: true, alias: "important" }, null: { pattern: i(/null|~/.source, "i"), lookbehind: true, alias: "important" }, string: { pattern: i(s), lookbehind: true, greedy: true }, number: { pattern: i(/[+-]?(?:0x[\da-f]+|0o[0-7]+|(?:\d+(?:\.\d*)?|\.\d+)(?:e[+-]?\d+)?|\.inf|\.nan)/.source, "i"), lookbehind: true }, tag: t, important: n, punctuation: /---|[:[\]{}\-,|>?]|\.\.\./ }, e.languages.yml = e.languages.yaml;
> }(Prism), Prism.languages.markup = { comment: { pattern: /<!--(?:(?!<!--)[\s\S])*?-->/, greedy: true }, prolog: { pattern: /<\?[\s\S]+?\?>/, greedy: true }, doctype: { pattern: /<!DOCTYPE(?:[^>"'[\]]|"[^"]*"|'[^']*')+(?:\[(?:[^<"'\]]|"[^"]*"|'[^']*'|<(?!!--)|<!--(?:[^-]|-(?!->))*-->)*\]\s*)?>/i, greedy: true, inside: { "internal-subset": { pattern: /(^[^\[]*\[)[\s\S]+(?=\]>$)/, lookbehind: true, greedy: true, inside: null }, string: { pattern: /"[^"]*"|'[^']*'/, greedy: true }, punctuation: /^<!|>$|[[\]]/, "doctype-tag": /^DOCTYPE/i, name: /[^\s<>'"]+/ } }, cdata: { pattern: /<!\[CDATA\[[\s\S]*?\]\]>/i, greedy: true }, tag: { pattern: /<\/?(?!\d)[^\s>\/=$<%]+(?:\s(?:\s*[^\s>\/=]+(?:\s*=\s*(?:"[^"]*"|'[^']*'|[^\s'">=]+(?=[\s>]))|(?=[\s/>])))+)?\s*\/?>/, greedy: true, inside: { tag: { pattern: /^<\/?[^\s>\/]+/, inside: { punctuation: /^<\/?/, namespace: /^[^\s>\/:]+:/ } }, "special-attr": [], "attr-value": { pattern: /=\s*(?:"[^"]*"|'[^']*'|[^\s'">=]+)/, inside: { punctuation: [{ pattern: /^=/, alias: "attr-equals" }, { pattern: /^(\s*)["']|["']$/, lookbehind: true }] } }, punctuation: /\/?>/, "attr-name": { pattern: /[^\s>\/]+/, inside: { namespace: /^[^\s>\/:]+:/ } } } }, entity: [{ pattern: /&[\da-z]{1,8};/i, alias: "named-entity" }, /&#x?[\da-f]{1,8};/i] }, Prism.languages.markup.tag.inside["attr-value"].inside.entity = Prism.languages.markup.entity, Prism.languages.markup.doctype.inside["internal-subset"].inside = Prism.languages.markup, Prism.hooks.add("wrap", function(e) {
|   "entity" === e.type && (e.attributes.title = e.content.replace(/&amp;/, "&"));
| }), Object.defineProperty(Prism.languages.markup.tag, "addInlined", { value: function(e, n) {
```

### Steps to reproduce

I'll see if I can come up with a repro

### Expected behavior

Build doesn't fail

### Actual behavior

Build fails - error related to webpack and Prism 

### Your environment

- Public source code: https://github.com/johnnyreilly/blog.johnnyreilly.com/pull/712
- Public site URL:https://johnnyreilly.com
- Docusaurus version used: 3.0.0-rc.1
- Environment name and version (e.g. Chrome 89, Node.js 16.4):
- Operating system and version (e.g. Ubuntu 20.04.2 LTS):

### Self-service

- [ ] I'd be willing to fix this bug myself.

**Comment by slorber** [comment]
Hey

You seem to have a custom prism dependency that is probably not supposed to be there.

Unfortunately I'll have very limited time in the next few days and won't be able to investigate.

Stable is planned for next week. Unless you can come up with a small repro and it's critical we well keep the release schedule

**Comment by johnnyreilly** [comment]
Which custom prism dependency do you mean?  I don't think I've got anything custom?

**Comment by slorber** [comment]
A brand new website doesn't have a prism dependency. I don't know why you have it there but it might be the problem.

**Comment by johnnyreilly** [comment]
It's a general issue @slorber.  Here's the repro:

```
npx create-docusaurus@3.0.0-rc.1 my-website classic
cd my-website
npm run build
```

**Comment by johnnyreilly** [comment]
> A brand new website doesn't have a prism dependency. I don't know why you have it there but it might be the problem.

You're right - I had that for historic reasons; when I needed to pin to a newer version of prism than was installed by default. I've removed it now.  I didn't need it anymore.  

But yeah, this seems to be a general issue.

**Comment by slorber** [comment]
🤔 CI runs exactly those steps and is able to build/typecheck/start on all OS.
I also ran those locally just after releasing no issue found.

I won't be able to investigate unfortunately, cf your DM. Please try with Docusaurus.new

**Comment by johnnyreilly** [comment]
I've just tested on a Mac and the same issue presents:

```
npx create-docusaurus@3.0.0-rc.1 my-website classic
cd my-website
npm run build

> my-website@0.0.0 build
> docusaurus build

[INFO] [en] Creating an optimized production build...

✖ Client
  Compiled with some errors in 18.98s

● Server █████████████████████████ building (65%) 1/1 entries 2297/2297 dependencies 683/683 modules 0 active 
 

[ERROR] Client bundle compiled with errors therefore further build is impossible.
Module parse failed: Unexpected token (763:929)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|     return "(?:" + r + "|" + s + ")";
|   })), lookbehind: true, greedy: true, alias: "atrule" }, directive: { pattern: /(^[ \t]*)%.+/m, lookbehind: true, alias: "important" }, datetime: { pattern: i(/\d{4}-\d\d?-\d\d?(?:[tT]|[ \t]+)\d\d?:\d{2}:\d{2}(?:\.\d*)?(?:[ \t]*(?:Z|[-+]\d\d?(?::\d{2})?))?|\d{4}-\d{2}-\d{2}|\d\d?:\d{2}(?::\d{2}(?:\.\d*)?)?/.source), lookbehind: true, alias: "number" }, boolean: { pattern: i(/false|true/.source, "i"), lookbehind: true, alias: "important" }, null: { pattern: i(/null|~/.source, "i"), lookbehind: true, alias: "important" }, string: { pattern: i(s), lookbehind: true, greedy: true }, number: { pattern: i(/[+-]?(?:0x[\da-f]+|0o[0-7]+|(?:\d+(?:\.\d*)?|\.\d+)(?:e[+-]?\d+)?|\.inf|\.nan)/.source, "i"), lookbehind: true }, tag: t, important: n, punctuation: /---|[:[\]{}\-,|>?]|\.\.\./ }, e.languages.yml = e.languages.yaml;
> }(Prism), Prism.languages.markup = { comment: { pattern: /<!--(?:(?!<!--)[\s\S])*?-->/, greedy: true }, prolog: { pattern: /<\?[\s\S]+?\?>/, greedy: true }, doctype: { pattern: /<!DOCTYPE(?:[^>"'[\]]|"[^"]*"|'[^']*')+(?:\[(?:[^<"'\]]|"[^"]*"|'[^']*'|<(?!!--)|<!--(?:[^-]|-(?!->))*-->)*\]\s*)?>/i, greedy: true, inside: { "internal-subset": { pattern: /(^[^\[]*\[)[\s\S]+(?=\]>$)/, lookbehind: true, greedy: true, inside: null }, string: { pattern: /"[^"]*"|'[^']*'/, greedy: true }, punctuation: /^<!|>$|[[\]]/, "doctype-tag": /^DOCTYPE/i, name: /[^\s<>'"]+/ } }, cdata: { pattern: /<!\[CDATA\[[\s\S]*?\]\]>/i, greedy: true }, tag: { pattern: /<\/?(?!\d)[^\s>\/=$<%]+(?:\s(?:\s*[^\s>\/=]+(?:\s*=\s*(?:"[^"]*"|'[^']*'|[^\s'">=]+(?=[\s>]))|(?=[\s/>])))+)?\s*\/?>/, greedy: true, inside: { tag: { pattern: /^<\/?[^\s>\/]+/, inside: { punctuation: /^<\/?/, namespace: /^[^\s>\/:]+:/ } }, "special-attr": [], "attr-value": { pattern: /=\s*(?:"[^"]*"|'[^']*'|[^\s'">=]+)/, inside: { punctuation: [{ pattern: /^=/, alias: "attr-equals" }, { pattern: /^(\s*)["']|["']$/, lookbehind: true }] } }, punctuation: /\/?>/, "attr-name": { pattern: /[^\s>\/]+/, inside: { namespace: /^[^\s>\/:]+:/ } } } }, entity: [{ pattern: /&[\da-z]{1,8};/i, alias: "named-entity" }, /&#x?[\da-f]{1,8};/i] }, Prism.languages.markup.tag.inside["attr-value"].inside.entity = Prism.languages.markup.entity, Prism.languages.markup.doctype.inside["internal-subset"].inside = Prism.languages.markup, Prism.hooks.add("wrap", function(e) {
|   "entity" === e.type && (e.attributes.title = e.content.replace(/&amp;/, "&"));
| }), Object.defineProperty(Prism.languages.markup.tag, "addInlined", { value: function(e, n) {
```

**Comment by maciejkopecpl** [comment]
So I see the same issue when I try to create a fresh app for 2.4.3
```
Module parse failed: Unexpected token (961:17)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|       'special-attr': [],
|       'attr-value': {
>         pattern: /=\s*(?:"[^"]*"|'[^']*'|[^\s'">=]+)/,
|         inside: {
|           'punctuation': [{
[ERROR] Client bundle compiled with errors therefore further build is impossible.
```

**Comment by johnnyreilly** [comment]
I also managed to replicate in GitHub Actions: https://github.com/johnnyreilly/blog.johnnyreilly.com/actions/runs/6659467622/job/18098597547?pr=712

I added this:

```bash
npx create-docusaurus@3.0.0-rc.1 my-website classic
cd my-website
npm run build
```

To one of my workflows and the output was:

```
[success] [webpackbar] Client: Compiled with some errors in 20.88s
Error:  Client bundle compiled with errors therefore further build is impossible.
Module parse failed: Unexpected token (763:929)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|     return "(?:" + r + "|" + s + ")";
|   })), lookbehind: true, greedy: true, alias: "atrule" }, directive: { pattern: /(^[ \t]*)%.+/m, lookbehind: true, alias: "important" }, datetime: { pattern: i(/\d{4}-\d\d?-\d\d?(?:[tT]|[ \t]+)\d\d?:\d{2}:\d{2}(?:\.\d*)?(?:[ \t]*(?:Z|[-+]\d\d?(?::\d{2})?))?|\d{4}-\d{2}-\d{2}|\d\d?:\d{2}(?::\d{2}(?:\.\d*)?)?/.source), lookbehind: true, alias: "number" }, boolean: { pattern: i(/false|true/.source, "i"), lookbehind: true, alias: "important" }, null: { pattern: i(/null|~/.source, "i"), lookbehind: true, alias: "important" }, string: { pattern: i(s), lookbehind: true, greedy: true }, number: { pattern: i(/[+-]?(?:0x[\da-f]+|0o[0-7]+|(?:\d+(?:\.\d*)?|\.\d+)(?:e[+-]?\d+)?|\.inf|\.nan)/.source, "i"), lookbehind: true }, tag: t, important: n, punctuation: /---|[:[\]{}\-,|>?]|\.\.\./ }, e.languages.yml = e.languages.yaml;
> }(Prism), Prism.languages.markup = { comment: { pattern: /<!--(?:(?!<!--)[\s\S])*?-->/, greedy: true }, prolog: { pattern: /<\?[\s\S]+?\?>/, greedy: true }, doctype: { pattern: /<!DOCTYPE(?:[^>"'[\]]|"[^"]*"|'[^']*')+(?:\[(?:[^<"'\]]|"[^"]*"|'[^']*'|<(?!!--)|<!--(?:[^-]|-(?!->))*-->)*\]\s*)?>/i, greedy: true, inside: { "internal-subset": { pattern: /(^[^\[]*\[)[\s\S]+(?=\]>$)/, lookbehind: true, greedy: true, inside: null }, string: { pattern: /"[^"]*"|'[^']*'/, greedy: true }, punctuation: /^<!|>$|[[\]]/, "doctype-tag": /^DOCTYPE/i, name: /[^\s<>'"]+/ } }, cdata: { pattern: /<!\[CDATA\[[\s\S]*?\]\]>/i, greedy: true }, tag: { pattern: /<\/?(?!\d)[^\s>\/=$<%]+(?:\s(?:\s*[^\s>\/=]+(?:\s*=\s*(?:"[^"]*"|'[^']*'|[^\s'">=]+(?=[\s>]))|(?=[\s/>])))+)?\s*\/?>/, greedy: true, inside: { tag: { pattern: /^<\/?[^\s>\/]+/, inside: { punctuation: /^<\/?/, namespace: /^[^\s>\/:]+:/ } }, "special-attr": [], "attr-value": { pattern: /=\s*(?:"[^"]*"|'[^']*'|[^\s'">=]+)/, inside: { punctuation: [{ pattern: /^=/, alias: "attr-equals" }, { pattern: /^(\s*)["']|["']$/, lookbehind: true }] } }, punctuation: /\/?>/, "attr-name": { pattern: /[^\s>\/]+/, inside: { namespace: /^[^\s>\/:]+:/ } } } }, entity: [{ pattern: /&[\da-z]{1,8};/i, alias: "named-entity" }, /&#x?[\da-f]{1,8};/i] }, Prism.languages.markup.tag.inside["attr-value"].inside.entity = Prism.languages.markup.entity, Prism.languages.markup.doctype.inside["internal-subset"].inside = Prism.languages.markup, Prism.hooks.add("wrap", function(e) {
|   "entity" === e.type && (e.attributes.title = e.content.replace(/&amp;/, "&"));
| }), Object.defineProperty(Prism.languages.markup.tag, "addInlined", { value: function(e, n) {
```

You can see the commit that adds it here: https://github.com/johnnyreilly/blog.johnnyreilly.com/pull/712/commits/3a6fef9a6a2c1b4d1a588615254f9a07267b7ff8 (It's not using my blog at all here; it was just the easiest way for me to make use of a GitHub Action)

**Comment by slorber** [comment]
Damn, I also see it now, while using the exact same command I used a few hours ago

```bash
npx --yes create-docusaurus@next new-release-test-website classic --typescript
cd new-release-test-website
yarn typecheck
yarn build
yarn start
```


I was able to build it:

<img width="589" alt="CleanShot 2023-10-26 at 23 19 17@2x" src="https://github.com/facebook/docusaurus/assets/749374/93859469-e2cf-4854-92b5-e0086891d188">

And I still can, but the newly initialized project can't build.

That may be a npm dependency that upgraded with a bug randomly? 🤔 weird coincidence

**Comment by maciejkopecpl** [comment]
@slorber can you run `npm ls --all` in the one that works for you and then in the one that doesn't? we might be able to track down the dependency that breaks it

**Comment by slorber** [comment]
[good2.zip](https://github.com/facebook/docusaurus/files/13183067/good2.zip)

Here's a rc.1 project that works for me. 
Not sure what changed exactly yet but it's weird.

**Comment by jprusik** [comment]
Just an additional data point - I'm also seeing this when doing `npx create-docusaurus@3.0.0-rc.1 del-test classic --typescript` (directory doesn't exist prior) and subsequent `npm run start` (I got the same result with rc.0)

**Comment by retiutut** [comment]
> I also managed to replicate in GitHub Actions. I took a workflow, commented out the code that was there and added this in its place:
> 
> ```shell
> npx create-docusaurus@3.0.0-rc.1 my-website classic
> cd my-website
> npm run build
> ```
> 
> To one of my workflows and the output was: https://github.com/johnnyreilly/blog.johnnyreilly.com/actions/runs/6659467622/job/18098597547?pr=712
> 
> ```
> [success] [webpackbar] Client: Compiled with some errors in 20.88s
> Error:  Client bundle compiled with errors therefore further build is impossible.
> Module parse failed: Unexpected token (763:929)
> You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
> |     return "(?:" + r + "|" + s + ")";
> |   })), lookbehind: true, greedy: true, alias: "atrule" }, directive: { pattern: /(^[ \t]*)%.+/m, lookbehind: true, alias: "important" }, datetime: { pattern: i(/\d{4}-\d\d?-\d\d?(?:[tT]|[ \t]+)\d\d?:\d{2}:\d{2}(?:\.\d*)?(?:[ \t]*(?:Z|[-+]\d\d?(?::\d{2})?))?|\d{4}-\d{2}-\d{2}|\d\d?:\d{2}(?::\d{2}(?:\.\d*)?)?/.source), lookbehind: true, alias: "number" }, boolean: { pattern: i(/false|true/.source, "i"), lookbehind: true, alias: "important" }, null: { pattern: i(/null|~/.source, "i"), lookbehind: true, alias: "important" }, string: { pattern: i(s), lookbehind: true, greedy: true }, number: { pattern: i(/[+-]?(?:0x[\da-f]+|0o[0-7]+|(?:\d+(?:\.\d*)?|\.\d+)(?:e[+-]?\d+)?|\.inf|\.nan)/.source, "i"), lookbehind: true }, tag: t, important: n, punctuation: /---|[:[\]{}\-,|>?]|\.\.\./ }, e.languages.yml = e.languages.yaml;
> > }(Prism), Prism.languages.markup = { comment: { pattern: //, greedy: true }, prolog: { pattern: /<\?[\s\S]+?\?>/, greedy: true }, doctype: { pattern: /<!DOCTYPE(?:[^>"'[\]]|"[^"]*"|'[^']*')+(?:\[(?:[^<"'\]]|"[^"]*"|'[^']*'|<(?!!--)|)*\]\s*)?>/i, greedy: true, inside: { "internal-subset": { pattern: /(^[^\[]*\[)[\s\S]+(?=\]>$)/, lookbehind: true, greedy: true, inside: null }, string: { pattern: /"[^"]*"|'[^']*'/, greedy: true }, punctuation: /^<!|>$|[[\]]/, "doctype-tag": /^DOCTYPE/i, name: /[^\s<>'"]+/ } }, cdata: { pattern: /<!\[CDATA\[[\s\S]*?\]\]>/i, greedy: true }, tag: { pattern: /<\/?(?!\d)[^\s>\/=$<%]+(?:\s(?:\s*[^\s>\/=]+(?:\s*=\s*(?:"[^"]*"|'[^']*'|[^\s'">=]+(?=[\s>]))|(?=[\s/>])))+)?\s*\/?>/, greedy: true, inside: { tag: { pattern: /^<\/?[^\s>\/]+/, inside: { punctuation: /^<\/?/, namespace: /^[^\s>\/:]+:/ } }, "special-attr": [], "attr-value": { pattern: /=\s*(?:"[^"]*"|'[^']*'|[^\s'">=]+)/, inside: { punctuation: [{ pattern: /^=/, alias: "attr-equals" }, { pattern: /^(\s*)["']|["']$/, lookbehind: true }] } }, punctuation: /\/?>/, "attr-name": { pattern: /[^\s>\/]+/, inside: { namespace: /^[^\s>\/:]+:/ } } } }, entity: [{ pattern: /&[\da-z]{1,8};/i, alias: "named-entity" }, /&#x?[\da-f]{1,8};/i] }, Prism.languages.markup.tag.inside["attr-value"].inside.entity = Prism.languages.markup.entity, Prism.languages.markup.doctype.inside["internal-subset"].inside = Prism.languages.markup, Prism.hooks.add("wrap", function(e) {
> |   "entity" === e.type && (e.attributes.title = e.content.replace(/&amp;/, "&"));
> | }), Object.defineProperty(Prism.languages.markup.tag, "addInlined", { value: function(e, n) {
> ```
> 
> You can see the commit that adds it here: [johnnyreilly/blog.johnnyreilly.com@3a6fef9](https://github.com/johnnyreilly/blog.johnnyreilly.com/commit/3a6fef9a6a2c1b4d1a588615254f9a07267b7ff8) (It's not using my blog source code at all here; it was just the easiest way for me to make use of a GitHub Action to test `create-docusaurus@3.0.0-rc.1`)
> 
> You can see the full code of the GitHub Action here: https://github.com/johnnyreilly/blog.johnnyreilly.com/blob/3a6fef9a6a2c1b4d1a588615254f9a07267b7ff8/.github/workflows/build-test-docusaurus.yml

I'm getting this exact error now from a branch that was working with Docusaurus 3.0.0-beta.0. 

```
  "name": "prism-react-renderer",
  "version": **"2.1.0",**
```

![image](https://github.com/facebook/docusaurus/assets/25914123/7358ad27-7b85-4049-8366-8f8d9ebf5ef4)

**Comment by berkaygiris** [comment]
I also get the same error using v2.4.3. This started to happen in a repo with no code change.

With a local modification suggested in https://github.com/facebook/docusaurus/issues/9023#issuecomment-1568574913, I  was able to collect the following error logs

```
errors: [
  {
    moduleIdentifier: '<my-paty>/docs/node_modules/prism-react-renderer/prism/index.js',
    moduleName: './node_modules/prism-react-renderer/prism/index.js',
    loc: '961:17',
    message: 'Module parse failed: Unexpected token (961:17)\n' +
      'You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders\n' +
      "|       'special-attr': [],\n" +
      "|       'attr-value': {\n" +
      `>         pattern: /=\\s*(?:"[^"]*"|'[^']*'|[^\\s'">=]+)/,\n` +
      '|         inside: {\n' +
      "|           'punctuation': [{"
  }
],
```

**Comment by Josh-Cena** [comment]
Folks, this is because of https://github.com/acornjs/acorn/issues/1253. It's neither because of prism-react-renderer nor Docusaurus nor Webpack. I'm going to close until that one gets fixed; in the meantime you may want to pin your Acorn version through your package manager resolution.

**Comment by edukisto** [comment]
```sh
npm i acorn@8.10.0
```

**Comment by pjmiravalle** [comment]
> Folks, this is because of [acornjs/acorn#1253](https://github.com/acornjs/acorn/issues/1253). It's neither because of prism-react-renderer nor Docusaurus nor Webpack. I'm going to close until that one gets fixed; in the meantime you may want to pin your Acorn version through your package manager resolution.

This worked for me by pinning [acorn](https://github.com/acornjs/acorn) to `8.10.0` (the most recent version before the recent bug). Thanks!

```jsonc
# package.json

"@docusaurus/core": "2.4.3",
"@docusaurus/preset-classic": "2.4.3",

# ...

"acorn": "8.10.0"
```

**Comment by zxhycxq** [comment]
> ```js
> acorn
> ```
I  use "@docusaurus/core": "2.1.0", the same error

**Comment by zxhycxq** [comment]
> I also get the same error(误差) using v2.4.3. This started to happen in a repo with no code(代码) change.
> 
> With a local modification(改性) suggested in [#9023 (comment)](https://github.com/facebook/docusaurus/issues/9023#issuecomment-1568574913), I was able(能) to collect the following(遵循) logserror(误差)
> 
> ```
> errors: [
>   {
>     moduleIdentifier: '<my-paty>/docs/node_modules/prism-react-renderer/prism/index.js',
>     moduleName: './node_modules/prism-react-renderer/prism/index.js',
>     loc: '961:17',
>     message: 'Module parse failed: Unexpected token (961:17)\n' +
>       'You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders\n' +
>       "|       'special-attr': [],\n" +
>       "|       'attr-value': {\n" +
>       `>         pattern: /=\\s*(?:"[^"]*"|'[^']*'|[^\\s'">=]+)/,\n` +
>       '|         inside: {\n' +
>       "|           'punctuation': [{"
>   }
> ],
> ```


I use "@docusaurus/core": "2.1.0", the same error

**Comment by slorber** [comment]
Thanks for investigating @Josh-Cena 🤗

I also suspected Acorn by seeing the lock file diff but couldn't be 100% sure.

Glad it's not a Docusaurus bug, hope it will be fixed for next week's release

**Comment by johnnyreilly** [comment]
With an older version of acorn in place I'm now happily building my site with 3.0.0-rc.1: https://github.com/johnnyreilly/blog.johnnyreilly.com/pull/712 - thanks for looking into this @Josh-Cena and @slorber!

**Comment by slorber** [comment]
Apparently it's fixed in Acorn 8.11.2 🙌

**Comment by jprusik** [comment]
Confirmed - same fresh install steps are now working 👍
